onsstove/technology.py

```diff
@@ -150,59 +148,38 @@ def mortality(self, specs_file, paf_0_alri, paf_0_copd, paf_0_lc, paf_0_ihd):
         paf_ihd = self.paf(rr_ihd, 1 - specs_file['clean_cooking_access'])
         paf_lc = self.paf(rr_lc, 1 - specs_file['clean_cooking_access'])
 
-        mort_alri_U = specs_file["Urban_HHsize"] * (paf_0_alri - paf_alri) * specs_file["Mort_ALRI"]
-        mort_copd_U = specs_file["Urban_HHsize"] * (paf_0_copd - paf_copd) * specs_file["Mort_COPD"]
-        mort_ihd_U = specs_file["Urban_HHsize"] * (paf_0_ihd - paf_ihd) * specs_file["Mort_IHD"]
-        mort_lc_U = specs_file["Urban_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
-
-        mort_alri_R = specs_file["Rural_HHsize"] * (paf_0_alri - paf_alri) * specs_file["Mort_ALRI"]
-        mort_copd_R = specs_file["Rural_HHsize"] * (paf_0_copd - paf_copd) * specs_file["Mort_COPD"]
-        mort_ihd_R = specs_file["Rural_HHsize"] * (paf_0_ihd - paf_ihd) * specs_file["Mort_IHD"]
-        mort_lc_R = specs_file["Rural_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
+        mort_alri = specs_file["Population_start_year"] * (paf_0_alri - paf_alri) * (specs_file["Mort_ALRI"]/100000)
+        mort_copd = specs_file["Population_start_year"] * (paf_0_copd - paf_copd) * (specs_file["Mort_COPD"]/100000)
+        mort_ihd = specs_file["Population_start_year"] * (paf_0_ihd - paf_ihd) * (specs_file["Mort_IHD"]/100000)
+        mort_lc = specs_file["Population_start_year"] * (paf_0_lc - paf_lc) * (specs_file["Mort_LC"]/100000)
 
         cl_copd = {1: 0.3, 2: 0.2, 3: 0.17, 4: 0.17, 5: 0.16}
         cl_alri = {1: 0.7, 2: 0.1, 3: 0.07, 4: 0.07, 5: 0.06}
         cl_lc = {1: 0.2, 2: 0.1, 3: 0.24, 4: 0.23, 5: 0.23}
         cl_ihd = {1: 0.2, 2: 0.1, 3: 0.24, 4: 0.23, 5: 0.23}
 
         i = 1
-        mort_U_vector = []
-        mort_R_vector = []
+        mort_vector = []
         while i < 6:
-            mortality_alri_U = cl_alri[i] * specs_file["VSL"] * mort_alri_U / (1 + specs_file["Discount_rate"]) ** (i - 1)
-            mortality_copd_U = cl_copd[i] * specs_file["VSL"] * mort_copd_U / (
+            mortality_alri = cl_alri[i] * specs_file["VSL"] * mort_alri_U / (1 + specs_file["Discount_rate"]) ** (i - 1)
+            mortality_copd = cl_copd[i] * specs_file["VSL"] * mort_copd_U / (
```

Comment:
I think here we need to remove U from `mort_alri_U`.

Reply:
yes